src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderErrorAlert.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
-export default function NoProvidersAlert({ sortMethod, typeOfCareName }) {
+export default function ProviderErrorAlert({ sortMethod, typeOfCareName }) {
```

Comment:
I think we got our wires crossed here. `NoProvidersAlert` is totally fine as a name. It's not an error, it's just an alert we show when there are no providers. `LoadProvidersAlert` is used when there's a backend error, so I thought it should have Error in the name.

Reply:
I misunderstood but should be sorted now.